include/rmm/mr/device/logging_resource_adaptor.hpp

```diff
@@ -98,6 +98,19 @@ class logging_resource_adaptor final : public device_memory_resource {
    */
   bool supports_get_mem_info() const noexcept override { return upstream_->supports_streams(); }
 
+  /**
+   * @brief Return the value of the environment variable RMM_LOG_FILE.
+   * 
+   * @throws `rmm::logic_error` if `RMM_LOG_FILE` is not set.
+   * 
+   * @return The value of RMM_LOG_FILE as `std::string`.
+   */
+  static std::string get_default_filename() {
```

Comment:
Is this ok as a static method of `logging_resource_adaptor`, or should I move it elsewhere?

Reply:
Seems OK to me.

Reply:
Make it private.

Reply:
Ok, made it private, but had to add `make_logging_adaptor` a friend function in exchange